code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -34,6 +34,10 @@
 		return
 	if (!( istype(M, /mob) ))
 		return
+	if(M_CLUMSY in user.mutations)
+		if(prob(50))
+			user << "<span class='notice'>Oops!</span>"
+			M = user
 	if (reagents.total_volume)
 		user << "<span class='notice'>You inject [M] with [src].</span>"
 		M << "<span class='warning'>You feel a tiny prick!</span>"

```

Comment:
You might want to change this because it'll be 

Oops!

You inject uristmcassistant

You feel a tiny prick.

Which is kinda weird, it should be something like "you inject yourself on accident!"